mlir-tensorrt/compiler/lib/Dialect/Plan/Transforms/AllocTensors.cpp

```diff
@@ -856,11 +856,13 @@ class AllocTensorsPass
       }
     }
 
-    // First rewrite public functions to conform to DPS style.
     IRRewriter rewriter(ctx);
-    if (failed(rewriteNotPrivateFuncsToDPS(rewriter, op))) {
-      op->emitError("Failed to convert non-private functions to DPS");
-      return signalPassFailure();
+    if (!useNonDPSCallConv) {
```

Comment:
Missing test for `plan-alloc-tensors`,

You'll need to add a `RUN` line in `test/Dialect/Plan/plan-alloc-tensors.mlir` where this option is invoked with `true` and add `CHECK` statements to show what is the desired result.